app/content/drill/[id]/leaderboard.js

```diff
@@ -124,51 +136,113 @@ export default function Leaderboard() {
       <Text style={{ fontSize: 18, alignSelf: "center", paddingTop: 15 }}>
         {prettyTitle[drillInfo.mainOutputAttempt]}
       </Text>
-      <List.Section>
+      <View
+        style={{
+          backgroundColor: themeColors.highlight,
+          borderRadius: 8,
+          padding: 16,
+          margin: 5,
+          width: "90%",
+          justifyContent: "space-between",
+          alignSelf: "center",
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "center",
+        }}
+      >
+        <View style={{ width: "15%" }}>
+          <Icon source="trophy-variant-outline" size={40} />
+        </View>
+        <View style={{ width: "60%" }}>
+          <Text
+            style={{
+              fontSize: 12,
+            }}
+          >
+            All Time Record
+          </Text>
+          <Text
+            style={{
+              fontSize: 20,
+              fontWeight: "bold",
+            }}
+          >
+            {allTimeInfo.currentRecord.name}
+          </Text>
+          <Text style={{ fontSize: 12 }}>
+            {formatDate(allTimeInfo.currentRecord.time)}
+          </Text>
+        </View>
+
+        <View style={{ width: "25%" }}>
+          <Text style={{ fontSize: 16 }}>
+            {numTrunc(allTimeInfo.currentRecord.value, true)}{" "}
+            {allTimeInfo.currentRecord.distanceMeasure}
+          </Text>
+        </View>
+      </View>
+
+      <List.Section
+        style={{
+          backgroundColor: themeColors.highlight,
+          margin: 5,
+          borderRadius: 5,
+        }}
+      >
         {orderedLeaderboard.map((userId, idx) => {
           const attempt = leaderboard[userId][mainOutputAttempt];
           return (
-            <Link
-              key={userId}
-              href={{
-                pathname: `${currentPath}/attempts/${attempt["id"]}`,
-              }}
-              asChild
-              style={{ paddingLeft: 20 }}
-            >
-              <List.Item
-                title={userInfo[userId] ? userInfo[userId]["name"] : "Unknown"}
-                left={() => (
-                  <View
-                    style={{
-                      flexDirection: "row",
-                      alignItems: "center",
-                    }}
-                  >
-                    <Text style={{ width: 30 }}>
-                      {leaderboardRanks[idx].toString()}.
-                    </Text>
-                    <Image
+            <View key={userId}>
+              <Link
+                key={userId}
+                href={{
+                  pathname: `${currentPath}/attempts/${attempt["id"]}`,
+                }}
+                asChild
+                style={{ paddingLeft: 20 }}
+              >
+                <List.Item
+                  title={
+                    userInfo[userId] ? userInfo[userId]["name"] : "Unknown"
```

Comment:
Should unknown be deleted instead?

Reply:
I didn't code this, I would assume it is there as a fail safe but idk